examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,64 @@
+"""
+=====================================
+Plotting points on a Map with WCSAxes
+=====================================
+
+This example demonstrates the plotting of point, line and arch in pixel coordinate,
+world coordinate, and SkyCoords respectively when plotting a map with WCSAxes.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates.utils import GreatArc
+
+###############################################################################
+# Firstly, We will start by creating a Map using an AIA 171 image.
+# Secondly, Let's first plot a line on the map by using coordinates in arcseconds.
+# The array below `xx` and `yy` are the x and y coordinates that define a from
+# the Sun center (at 0, 0) to the point (500, 500) in arcsecs.
+# The standard plotting in WCSAxes expects pixel coordinates, but we can plot
+# world coordinates (i.e. arcsec) by using the `transform` keyword.
+# Note that the coordinates need to be in degrees rather than arcseconds.
```

Comment:
Move this down to be above the plot command?

Reply:
Sure Nabil, I did this as per my interaction with David. https://github.com/sunpy/sunpy/pull/4867/files#r593114980
But this makes sense will do it.

Reply:
I mean the last line of this block.

Reply:
So sorry i got it lately and delete the comment. Didn't see your reply. 
I have make the changes.